main/modes/fighter/fighter_menu.c

```diff
@@ -381,7 +381,14 @@ void fighterButtonCb(buttonEvt_t* evt)
             // Pass button events from the Swadge mode to the menu
             if(evt->down)
             {
-                meleeMenuButton(fm->menu, evt->button);
+                if (evt->button == BTN_B && fm->menu->title != str_swadgeBros)
```

Comment:
This is very elegant, but also misleading in that a string can be passed to the callback that wasn't actually in the menu. Did you check that all fighter menu callbacks handle `str_back`?

Reply:
Yeah, I checked and conveniently every menu callback (other than the main menu) in fighter wisely includes an else clause with a "shouldn't happen, return just in case" clause.